libraries/kotlin.test/junit5/src/main/kotlin/JUnitSupport.kt

```diff
@@ -32,6 +32,16 @@ object JUnit5Asserter : Asserter {
         Assertions.assertEquals(expected, actual, message)
     }
 
+    @SinceKotlin("1.5")
+    override fun assertEquals(message: String?, expected: Double, actual: Double, absoluteTolerance: Double) {
+        Assertions.assertEquals(expected, actual, absoluteTolerance, message)
```

Comment:
> The implementation in this PR works the junit5 way, except it considers 0.0 as a validabsoluteTolerance value.

How is it possible if this override delegates to junit5 and therefore rejects 0.0?

Reply:
I mean the default implementation considers `0.0` as a valid `absoluteTolerance` value.
By the way, junit5 doesn't have `assertNotEquals` for floating point numbers, resulting to different behavior and different failure message style for `assertEquals` and `assertNotEquals`. 
Would removing the overrides be better for junit5?

Reply:
I think in this case it's better to have consistency of the contract.

Reply:
The supporting frameworks do not override the behavior now.